apps/communications/contacts/elements/settings.html

```diff
@@ -34,28 +34,6 @@ <h1 data-l10n-id="settings"></h1>
               </button>
             </li>
           </ul>
-          <gaia-subheader id="fb-header" skin="organic">
```

Comment:
The file style/fb_extensions.css has a rule for this fb-header, either we can remove it now, or do a step where we remove all fb_extensions.css file.

I've see Jorge creating specific css files just with the stuff needed being extracted from the fb mess, for me that sounds good.